content/en/docs/helm/helm_install.md

```diff
@@ -50,6 +50,8 @@ set for a key called 'foo', the 'newbar' value would take precedence:
 To check the generated manifests of a release without installing the chart,
 the '--debug' and '--dry-run' flags can be combined.
 
+Any chart's CRDs manifests can't be validated by `--dry-run` flag, according to [Helm's CRDs caveats](docs/chart_best_practices/custom_resource_definitions.md#some-caveats-and-explanations)
```

Comment:
Any chart's CRDs manifests can't be validated by `--dry-run` flag, according to [Helm's CRDs caveats](docs/chart_best_practices/custom_resource_definitions.md#some-caveats-and-explanations) --> CRD manifests in the `crds/` directory can't be validated by `--dry-run` flag. See [Helm's CRDs caveats](docs/chart_best_practices/custom_resource_definitions.md#some-caveats-and-explanations) for more details.

Reply:
@hickeyma many thanks on this request changes! Just updated the PR.